src/Collect/Support/Optional.php

```diff
@@ -0,0 +1,110 @@
+<?php
```

Comment:
Does this provide anything we don't have in the current version?

Reply:
Also removed.

Reply:
Grazi!